### ISSUE TITLE
Missing accessibility lable 

### ISSUE THREAD

**Issue by TStArK1233** [SimpleMobileTools/Simple-Camera#415]
![IMG_20230526_205800](https://github.com/SimpleMobileTools/Simple-Camera/assets/133884373/c03760c7-fca6-4a5d-8a8e-34e686590aa4)

The icons marked in white are not labled. Thank you for fixing it!😊😊

**Comment by tibbi** [comment]
move it into General-Discussion repo at https://github.com/SimpleMobileTools/General-Discussion/issues please as it affects all apps
